rasa/nlu/server.py

```diff
@@ -1,28 +1,49 @@
 import argparse
 import asyncio
-import io
 import logging
 import os
 from functools import wraps
-
+from inspect import isawaitable
+from sanic import Sanic, response
+from sanic.request import Request
+from sanic_cors import CORS
+from typing import Any, Callable, Optional, Text
 import simplejson
-from klein import Klein
-from twisted.internet import reactor, threads
-from twisted.internet.defer import inlineCallbacks, returnValue
 
 import rasa
 import rasa.utils.io
-from rasa.nlu import config, utils
+import rasa.utils.common
+import rasa.utils.endpoints
+from rasa.nlu import config, utils, constants
 import rasa.nlu.cli.server as cli
 from rasa.nlu.config import RasaNLUModelConfig
 from rasa.nlu.data_router import DataRouter, InvalidProjectError, MaxTrainingError
 from rasa.constants import MINIMUM_COMPATIBLE_VERSION
 from rasa.nlu.train import TrainingException
-from rasa.nlu.utils import json_to_string, read_endpoints
+from rasa.nlu.utils import read_endpoints
 
 logger = logging.getLogger(__name__)
 
 
+class ErrorResponse(Exception):
+    def __init__(self, status, reason, message, details=None, help_url=None):
+        self.error_info = {
+            "version": rasa.__version__,
+            "status": "failure",
+            "message": message,
+            "reason": reason,
+            "details": details or {},
+            "help": help_url,
+            "code": status,
+        }
+        self.status = status
+
+
+def _docs(sub_url: Text) -> Text:
```

Comment:
should this go to utils?

Reply:
It is kind of bound to the package, as the DOCS_BASE_URL differs for core and nlu. But will probably change anyway once we actually merge the servers.